lib/node_modules/@stdlib/stats/base/meanpn/lib/accessors.js

```diff
@@ -0,0 +1,75 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var gsumpw = require( '@stdlib/blas/ext/base/gsumpw' ).ndarray;
+var gapxsumpw = require( '@stdlib/blas/ext/base/gapxsumpw' ).ndarray;
+
+
+// MAIN //
+
+/**
+* Computes the arithmetic mean of a strided array using a two-pass error correction algorithm.
+* @private
```

Comment:
```suggestion
*
* @private
```

Always include a blank line between the description and tags.